events/views.py

```diff
@@ -219,3 +225,22 @@ def stats(request):
 
     return render_to_response(
         'stats/index.html', data, context_instance=RequestContext(request))
+
+def events_category(request, id, slug):
+    """
+        Filter evets for category
+    """
+    try:
+        events = CategoryEvent.objects.filter(category_id=id)
```

Comment:
Here you are getting the category object, why do you assign it to a variable called ``events``? I would query for events using this category ID.